packages/block-editor/src/components/block-preview/index.js

```diff
@@ -70,6 +70,7 @@ export function BlockPreview( {
 		<ExperimentalBlockEditorProvider
 			value={ renderedBlocks }
 			settings={ settings }
+			inert={ true }
```

Comment:
There's probably more that we can eventually do for "inert" editors, but this is a good start. @youknowriad I'd love your opinion here. I'm also unsure about the naming. Maybe inert makes it sound like you can't interact with it, but you'd need to fix that yourself.

Reply:
We already have a `__unstableIsPreviewMode: true` flag, we shouldn't be introducing a new one. For me it's just the same as "inert" prop here.

Reply:
This is even better :)